gensim/sklearn_api/w2vmodel.py

```diff
@@ -83,4 +83,5 @@ def transform(self, words):
         return np.reshape(np.array(X), (len(words), self.size))
 
     def partial_fit(self, X):
-        raise NotImplementedError("'partial_fit' has not been implemented for W2VTransformer")
+        raise NotImplementedError("'partial_fit' has not been implemented for W2VTransformer since 'update()' function for Word2Vec class is experimental. "
```

Comment:
@piskvorky The original error message was not correct (since there is no `update()` function in `Word2Vec` class as you correctly pointed out) and I have updated the message now. My apologies for the confusion.
This change was made in reference to [this comment](https://github.com/RaRe-Technologies/gensim/pull/1437#discussion_r123597260) by @gojomo in one of the older PRs.